projects/step-core/src/lib/client/generated/services/EntitiesService.ts

```diff
@@ -0,0 +1,96 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import { Injectable } from '@angular/core';
+import type { Observable } from 'rxjs';
+
+import { BaseHttpRequest } from '../core/BaseHttpRequest';
+
+@Injectable({providedIn:'root'})
+export class EntitiesService {
+
+    constructor(public readonly httpRequest: BaseHttpRequest) {}
+
+    /**
+     * @param type
+     * @param id
+     * @returns any default response
+     * @throws ApiError
+     */
+    public get(
+        type: string,
+        id: string,
+    ): Observable<any> {
+        return this.httpRequest.request({
+            method: 'GET',
+            url: '/entities/{type}/{id}',
+            path: {
+                'type': type,
+                'id': id,
+            },
+        });
+    }
+
+    /**
+     * @param type
+     * @param id
+     * @returns any default response
+     * @throws ApiError
+     */
+    public delete1(
+        type: string,
+        id: string,
+    ): Observable<any> {
+        return this.httpRequest.request({
+            method: 'DELETE',
+            url: '/entities/{type}/{id}',
+            path: {
+                'type': type,
+                'id': id,
+            },
+        });
+    }
+
+    /**
+     * @param type
+     * @param requestBody
+     * @returns any default response
+     * @throws ApiError
+     */
+    public find1(
+        type: string,
+        requestBody?: Record<string, string>,
+    ): Observable<any> {
+        return this.httpRequest.request({
+            method: 'POST',
+            url: '/entities/{type}/find',
+            path: {
+                'type': type,
+            },
+            body: requestBody,
+            mediaType: 'application/json',
+        });
+    }
+
+    /**
+     * @param type
+     * @param requestBody
+     * @returns any default response
+     * @throws ApiError
+     */
+    public save1(
```

Comment:
Such methods with a number at the end of the name, still remain. Should we remove it, or will it be done later?

Reply:
Are there still methods in the API?

As an explanation, the numbers are added by swagger when a method name is used multiple times (i.e. delete for /plan/ and for /keaywords/) but David tried to remove the functions and I did a quick check and it looked good. If there are still numbers in the API we should ask him to rename the functions.

Reply:
when opening https://stepos-master.exense.ch/rest/private-openapi.json
and searching for `_1` I can still find 4 occurences of such method names, one of them beeing `save_1`, resulting in the generated code you mentioned above. So the generator did all good, the flaw is still in the API-specs.. but there are only 4 remaining :dancers: :dancers:

Reply:
I suggest to merge the branch. Sooner or later we will have to regenerate again anyways due to API changes